<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioView</class>
 <widget class="QMainWindow" name="AudioView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(52,51,64);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_File_Name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Sans</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color : rgb(255,255,255);
	border : 2px solid rgb(255,255,255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>FILE NAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal
{
	height : 10px;
	width : 600px;
	background : red;
	border-radius : 5px;
}
QSlider::handle:horizontal
{
	background : rgb(255,255,255);
	width : 20px;
	height : 20px;
	margin : -7px -7px;
	border-radius : 10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volume">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal
{
	height : 10px;
	width : 500px;
	background : red;
	border-radius : 5px;
}
QSlider::handle:horizontal
{
	background : rgb(255,255,255);
	width : 20px;
	height : 20px;
	margin : -7px -7px;
	border-radius : 10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border : none;
	border-radius : 15px;
	min-width : 100px;
	max-width : 100px;
	min-height : 100px;
	max-height: 100px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	padding : 0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Current_Time">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>165</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Sans</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	min-height : 40px;
	max-height: 40px;
	color : rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>165</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	min-height : 40px;
	max-height: 40px;
	color : rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>801</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Backward">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border : none;
	border-radius : 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	min-width : 120px;
	max-width : 120px;
	min-height : 120px;
	max-height: 120px;
	padding : 0px;
}

QPushButton:Hover
{
	background-color : rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Play_Pause">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border : none;
	border-radius : 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	min-width : 120px;
	max-width : 120px;
	min-height : 120px;
	max-height: 120px;
	padding : 0px;
}

QPushButton:Hover
{
	background-color : rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border : none;
	border-radius : 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	min-width : 120px;
	max-width : 120px;
	min-height : 120px;
	max-height: 120px;
	padding : 0px;
}

QPushButton:Hover
{
	background-color : rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Forward">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border : none;
	border-radius : 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	min-width : 120px;
	max-width : 120px;
	min-height : 120px;
	max-height: 120px;
	padding : 0px;
}

QPushButton:Hover
{
	background-color : rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen_Audio">
   <property name="text">
    <string>Open Audio</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
